examples/mixed-test/SwiftObj.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  mixed_test.swift
+//  mixed-test
+//
+//  Created by Oscar Bonilla on 7/23/18.
+//  Copyright © 2018 Oscar Bonilla. All rights reserved.
+//
+
+public class SwiftObj : NSObject {
```

Comment:
needs an `import Foundation`

Reply:
Shouldn't that come from the Objective-C headers?

Reply:
I don't think Swift will pull in transitive modules like that?